doc/installation_and_administration_guide/configuration.md

```diff
@@ -310,7 +312,59 @@ cygnusagent.channels.mkafka-channel.transactionCapacity = 100
 
 [Top](#top)
 
-##<a name="section4"></a>`log4j.properties`
+##<a name="section4"></a>`flume-env.sh`
+The file `flume-env.sh`  can be instantiated from a template given in the Cygnus repository, `conf/flume-env.sh.template`. 
+
+```
+#=============================================
+# To be put in APACHE_FLUME_HOME/conf/flume-env.sh
+#=============================================
+
+#JAVA_HOME=/usr/lib/jvm/java-6-sun
+
+# Give Flume more memory and pre-allocate, enable remote monitoring via JMX
+#JAVA_OPTS="-Xms100m -Xmx200m -Dcom.sun.management.jmxremote"
+
+# Note that the Flume conf directory is always included in the classpath.
+#FLUME_CLASSPATH="/path/to/the/flume/classpath"
+```
+`flume-env.sh` file has been inherited from Apache Flume, and it is used in order to configure certain Flume parameters such as the classpath, some Java opcions...
```

Comment:
such as the classpath -> such as an alternative classpath
opcions -> options
... -> etc.